src/app/views/infoblock/infoblock.js

```diff
@@ -0,0 +1,48 @@
+class InfoblockController {
+
+  /** @ngInject */
+  constructor($log) {
+    this.$log = $log;
+  }
+
+  $onChanges() {
+    if (!this.variable) {
+      return;
+    }
+
+    this.value = this.variable.value;
+    this.title = this.variable.title;
+
+    this.colorClass = '';
+    if (this.colorOverride) {
+      this.colorClass = `color-${this.colorOverride}`;
+    } else if (this.isChange && this.value !== 0) {
+      let ispositive = (this.value > 0);
+      if (this.reverseColor) {
+        ispositive = !ispositive;
+      }
+      this.colorClass = ispositive ? 'color-positive' : 'color-negative';
+    }
+
+    this.value = Math.round(this.value * 100) / 100;
```

Comment:
Ugh. Yeah that's nasty for the number filter. Would require rolling your own. Does it still add the extra zero if you preformat the number you send to it to only include 2 decimal places?

I'm not sure I can answer this atm. The two locales we're most interested in for this app are en and mn. It may be that we would need to roll a custom filter that checks the set language via ng-translate and modifies the symbols accordingly. For now I think this piece is fine as is.